tests/transforms/test_convert_rois.py

```diff
@@ -1,56 +1,38 @@
 import pytest
 import json
-import os
-
 import numpy as np
-from marshmallow.validate import ValidationError
+from ophys_etl.transforms.convert_rois import (
+        BinarizerAndROICreator, LIMSCompatibleROIFormat)
+from ophys_etl.transforms.roi_transforms import StandardROI
 
-from ophys_etl.transforms.convert_rois import BinarizerAndROICreator
 
+def test_output_schema_element():
```

Comment:
Nice optimization here!

Reply:
> Nice optimization here!

It's not true validation, but, this actually already kept me honest when I rebased and you had removed `cell_specimen_id`.
I thought about [pydantic](https://pydantic-docs.helpmanual.io/) but decided not to open that box right now.

Reply:
At this point we should just use a typed language... 😉

Reply:
I've been looking for a reason to try [Rust](https://www.rust-lang.org/)